### DISCUSSION TITLE
`Clone` should carry over the static part of UiFormatter

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5596] [Feature Requests]
For a UI formatter like `YEAR-######`, if YEAR has autonumbering turned off, the `Clone` button should clone the `YEAR` portion of the formatter, while resetting the parts that have autonumbering on (in this case `######` part)

Another example can be seen in the `Wesleyan paleobiology` database.
Formatter:
<img width="1800" alt="Screen Shot 2022-06-15 at 21 44 42" src="https://user-images.githubusercontent.com/40512816/174202573-a58f74e7-184b-4d87-b2ec-e673dfd18769.png">

Form:
<img width="927" alt="Screen Shot 2022-06-16 at 20 10 53" src="https://user-images.githubusercontent.com/40512816/174202601-86df0fb7-2034-4aa1-9b61-22655a5f7bd0.png">

In this case, the middle part `FVE` is not carried over and is instead replaced with `XXX`

**Comment by grantfitzsimmons** [comment]
@maxpatiiuk Still something to do?

**Comment by maxpatiiuk** [comment]
Not implemented yet. Would be useful.
I don't remember if sp6 has something like this, but without doubt sp7 is still not on par with sp6 as far as field formatters
